bears/python/PEP8Bear.py

```diff
@@ -49,6 +49,6 @@ def run(self, filename, file,
 
         for diff in diffs:
             yield Result(self,
-                         "The code does not comply to PEP8.",
+                         "The code does not comply to PyCodeStyle.",
```

Comment:
Removing the commit for now thought
Could you close the related issue with wont fix if you feel so ?